src/Command.ts

```diff
@@ -334,6 +334,15 @@ export default abstract class Command {
   }
 
   protected handleRejectedODataPromise(res: any): void {
+    if (cli.optionsFromArgs?.options.debug) {
```

Comment:
Let's get this value from `this.debug`, which is available in the Command class.

Reply:
Hi @waldekmastykarz, using `this.debug` generates a looooooot of printed errors when executing `npm run test` 😅 .

Do you want to use `this.debug` or stay with `cli.optionsFromArgs?.options.debug` here?

Reply:
Oh interesting! Are these errors intentional or are they result of broken tests that we should fix?

Reply:
The difference is that a lot of tests that throw an error intentionally with `assert.reject`, use `debug: true`. when we use `this.debug` in the `handleRejectedODataPromise`, this value is true when defining debug to true in the test.
When we use `cli.optionsFromArgs?.options.debug`, the `optionsFromArgs` is undefined and thus debug being undefined, which leads to the fact that the error stack trace isn't logged when testing

Reply:
Got it. I suggest that we make it more explicit that it's meant to exclude the output from tests. While `cli.optionsFromArgs?.options.debug` might do the job, I suspect that few months down the road, we'll have a hard time understanding why it's there and there's always a chance of the behavior changing. So instead, I suggest that we look into the ability to detect if we're in a test runner (I think there might be an env var being set by the test runner) and exclude the output if we are.

Reply:
That would be a great idea! It took me a while as I tried different scenario's:
- `process.env.NODE_ENV` is undefined
- I tried to set `NODE_ENV` to test in the scripts section of `package.json` no luck 😞 
- Looked into `process.env`, no env var set there
- Looked into `process`, no env var set there
- Looked into the `test` var (`Mocha.TestFunction`), undefined

Eventually landed with `global.it`, which seems to be used in the CLI code already 😄

Reply:
Thorough research! Very nice! `global.it` seems like a good choice. 👏